bleak/backends/bluezdbus/client.py

```diff
@@ -575,6 +581,9 @@ def mtu_size(self) -> int:
     async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
         """Get all services registered for this GATT server.
 
+        Keyword Args:
+            timeout (float): Defaults to 10.0.
```

Comment:
Perhaps specify the unit as seconds; so "timeout (float): Defaults to 10.0 seconds."